samples/simpleasset/mocks/chaincodestub.go

```diff
@@ -0,0 +1,2878 @@
+// Code generated by counterfeiter. DO NOT EDIT.
```

Comment:
This looks like a copy of the mock stub files in the Fabric Interop CC? We shouldn't have multiple copies of the same file in different places in a repo.

I'd suggest creating a makefile and adding copy instructions there, just like we do for the protos.

Reply:
Okay, actually it was a copy from fabric-samples repo, I didn't realise it was present in fabric-interop-cc as well.
It was only for the purpose of testing the chaincode using `go test` command. 
Do you think using makefile for that is better? If yes I can do that.